### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by mlaventure**
rock

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,7 +96,13 @@ be found.
 - Fix an issue where Authz plugin would corrupt the payload body when faced with a large amount of data ([#20602](https://github.com/docker/docker/pull/20602))
 
 ### Runtime
-
+- Fix a panic that could occur when cleanup after a container started with invalid parameters [#21716](https://github.com/docker/docker/pull/21716)
+- Fix a race with event timers stopping early [#21692](https://github.com/docker/docker/pull/21692)
+- Fix race conditions in the layer store, potentially corrupting the map and crashing the process [#21677](https://github.com/docker/docker/pull/21677)
+- Un-deprecate auto-creation of host directories for mounts. This feature was marked deprecated in [#21666](https://github.com/docker/docker/pull/21666)
```
Comment (thaJeztah) [inline CHANGELOG.md]: you need to add two spaces at the end of this line to force a line-break in the rendered Markdown

File: CHANGELOG.md [hunk 2]
```diff
@@ -91,7 +96,13 @@ be found.
 - Fix an issue where Authz plugin would corrupt the payload body when faced with a large amount of data ([#20602](https://github.com/docker/docker/pull/20602))
 
 ### Runtime
-
```
Comment (thaJeztah) [inline CHANGELOG.md]: can you put the blank line back?

**Comment by mlaventure** [comment]
Updated per @thaJeztah comments

**Comment by thaJeztah** [comment]
LGTM, thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -36,6 +37,7 @@ be found.
 
 ### Distribution
 
+- Fix the download manager closing the tempfile twice [#21676](https://github.com/docker/docker/pull/21676)
```
Comment (tiborvass) [inline CHANGELOG.md]: is this phrased weirdly or is it just me?

**Comment by thaJeztah** [comment]
carried in https://github.com/docker/docker/pull/21827 (missing parentheses)

**Pull request closed without merging**